style2.css

```diff
@@ -33,6 +116,21 @@ body {
   line-height: 30px;
   word-wrap: break-word;
   width: 400px;
+  margin-left: 30px;
+  color: #172b4d;
+}
+
+.para-box {
+  width: 1200px;
+  margin-left: 260px;
+}
+
+.paragraph2 {
```

Comment:
- [ ] Please correct the overlapping of text as required by the previous reviewer. Stick to the Figma design is a project requirement.
Your portfolio:
<img width="585" alt="Screen Shot 2022-08-27 at 7 51 40 PM" src="https://user-images.githubusercontent.com/47673311/187050844-aed99706-3d74-4db0-87f0-34d9de3a819b.png">

Figma template:
<img width="694" alt="Screen Shot 2022-08-27 at 7 51 59 PM" src="https://user-images.githubusercontent.com/47673311/187050855-ca88e5fc-c8c5-4758-a504-f8d3dd140444.png">

Reply:
- [ ] Kindly address this 👍🏻